### DISCUSSION TITLE
How to debug inside the notebook (inside VS Code)?

### DISCUSSION THREAD

**Discussion by adotname** [yunabe/tslab#60] [Q&A]
Hi,
Currently I have *tslab* installed, and can run cells just fine inside VS Code,
However when I set a debug breakpoint and then press "Run and Debug", some process runs then gives up.
Is this functionality yet unsupported, or is there some configuration required?

**Comment by aecorredor** [comment]
@adotname how were you able to configure tslab to work within VSCode? For me it works inside Jupyter Lab, but not inside VSCode.
